packages/bpk-docs/src/pages/NativeSpinnerPage/NativeSpinnerPage.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import BpkLink from 'bpk-component-link';
+import readme from 'react-native-bpk-component-spinner/readme.md';
+
+import iosScreenshotDefault from 'react-native-bpk-component-spinner/screenshots/ios/default.png';
+import iosScreenshotSmall from 'react-native-bpk-component-spinner/screenshots/ios/small.png';
+import androidScreenshotDefault from 'react-native-bpk-component-spinner/screenshots/android/default.png';
+import androidScreenshotSmall from 'react-native-bpk-component-spinner/screenshots/android/small.png';
+
+import Paragraph from './../../components/Paragraph';
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+
+import { THEMING } from './../../constants/routes';
+
+const reactNativeUrl = 'https://facebook.github.io/react-native/docs/activityindicator.html';
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb: 'By default, spinners come in a large size and are available in three colours: blue-500, gray-700, and white.', // eslint-disable-line max-len
```

Comment:
This is taken verbatim from the jira card but I'm wondering if we should say 'primary' 'dark' and 'light' instead of naming the tokens directly.